modules/nobidBidAdapter.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: bidder
+title: Nobid
+description: Prebid Nobid Bidder Adaptor
+biddercode: nobid
+hide: true
+media_types: banner
+gdpr_supported: true
+---
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name          | Scope    | Description | Example | Type     |
+|---------------|----------|-------------|---------|----------|
+| `siteId` | required | siteId is provided by your Nobid account manager |         | `integer` |
+
+# Test Parameters
+```
+    var adUnits = [
+        {
+            code: 'test-div',
+            mediaTypes: {
+                banner: {
+                    sizes: [[300, 250]],  // a display size
+                }
+            },
+            bids: [
+                {
+                    bidder: "nobid",
+                    params: {
+                        siteId: <SITE_ID_PROVIDED_BY_YOUR_NOBID_ACCOUNT_MANAGER>
+                    }
+                }
+            ]
+        },{
+            code: 'test-div',
+            mediaTypes: {
+                banner: {
+                    sizes: [[320, 50]],   // a mobile size
+                }
+            },
+            bids: [
+                {
+                    bidder: "nobid",
+                    params: {
+                        siteId: <SITE_ID_PROVIDED_BY_YOUR_NOBID_ACCOUNT_MANAGER>
```

Comment:
Thanks for submitting this.

However every bidders markdown is required to have valid Test ad units which will consistently return valid bids in order to be tested by PR reviewers as well as users.

Please see the [Required Files](http://prebid.org/dev-docs/bidder-adaptor.html#required-files) Section in the Docs for more information on the requirements on adding a New Bid Adapter.